Okta/OktaAuth.swift

```diff
@@ -36,7 +36,9 @@ public struct OktaAuthorization {
                 )
 
                 // Start the authorization flow
-                OktaAuth.currentAuthorizationFlow = OIDAuthState.authState(byPresenting: request, presenting: view){
+                let externalUserAgent = OktaExternalUserAgentIOS(presenting: view)
+                OktaAuth.currentAuthorizationFlow = OIDAuthState.authState(byPresenting: request, externalUserAgent: externalUserAgent)
+                {
```

Comment:
nit: Curly braces should be consistent throughout files. Lets move this up to line `40`

Reply:
Done.